lib/logger.js

```diff
@@ -5,15 +5,17 @@
 const bunyan = require('bunyan'),
       {LoggingBunyan} = require('@google-cloud/logging-bunyan');
 
-const loggingBunyan = new LoggingBunyan();
-const logger = bunyan.createLogger({
-  name: 'wpt-pr-bot',
-  streams: [
-    // Log to the console at 'info' and above
-    {stream: process.stdout, level: 'info'},
-    // And log to Stackdriver Logging, logging at 'debug' and above
-    loggingBunyan.stream('debug'),
-  ],
-});
+const streams = [];
+
+if (process.env.GOOGLE_APPLICATION_CREDENTIALS) {
+  // And log to Stackdriver Logging, logging at 'debug' and above
+  const loggingBunyan = new LoggingBunyan();
+  streams.push(loggingBunyan.stream('debug'));
```

Comment:
Is the plain stdout logger never useful in this case?

Reply:
I don't think so, no. It would end up somewhere in the logging console, but would be a duplicate of the structured log also provided. There shouldn't be other stuff in stdout that one needs to determine the order of logs relative to, the only use of console.log is in dead code, AFAICT.